src/utils/fits.rs

```diff
@@ -1,5 +1,5 @@
-use flate2::read::GzDecoder;
-use std::io::Read;
+use zune_inflate::DeflateDecoder;
```

Comment:
I should add a comment about this. It's basically an implementation of gzip/deflate that is faster than flate2. Plus it comes with a few handy options such as skipping checksum confirmation (ZTF should be sending us valid images anyways), and also we can provide it size hints to avoid it from reallocating memory as the decoded data grows larger (we can give it the max size in bytes we would get for a fits file if the image is at 63x63).

Reply:
done